relay/sources/relayd/fuzz/Cargo.toml

```diff
@@ -7,22 +7,17 @@ publish = false
 [package.metadata]
```

Comment:
Can you tell me why these changes were needed?

Reply:
this is all just updating an old fuzz harness that didn't compile due to name changes and API changes. the Cargo.toml file above this one declares the package name to "rudder-relayd" so that had to be updated. The libfuzzer-sys dep and harness style also were the old style, so I simplified it